.appveyor.yml

```diff
@@ -130,6 +100,36 @@ build_script:
   - ps: cp C:\projects\mlpack\build\include\mlpack C:\projects\mlpack\dist\win-installer\staging -recurse
   - ps: cp C:\projects\mlpack\doc\examples C:\projects\mlpack\dist\win-installer\staging -recurse
   
+  # Checking current gitversion or mlpack version.
+  - ps: >
+      if (Test-Path "C:\projects\mlpack\src\mlpack\core\util\gitversion.hpp")
+      {
+        # Converting string to x.x.x.x format.
+        $ver = (Get-Content "C:\projects\mlpack\src\mlpack\core\util\gitversion.hpp");
+        $gitversion = ($ver.Split('"')[1].Split(' ')[1].Split('-')[1]) + '0';
+        for ($i=0; $i -lt 4; $i++)
+        {
+          $env:MLPACK_VERSION += ([convert]::toint16($gitversion.substring($i*2, 2), 16)).ToString();
+          if ($i -lt 3)
+          {
+            $env:MLPACK_VERSION += '.'
+          }
+        }
+      }
```

Comment:
This code extracts the git version and modifies it to `x.x.x.x` format, if the automatically generated file `C:\projects\mlpack\src\mlpack\core\util\gitversion.hpp` exists.

Reply:
@KimSangYeon-DGU I wrote a clarification in #1506, can you try implementing that please instead?  I've pasted the comment below:

"Hehe, @KimSangYeon-DGU, thanks for the extensive analysis. But I was thinking something much simpler, just like we do for existing Linux builds. :) Basically we just use the git hash... so if the hash is b23d5ce then the version is mlpack git-b23d5ce. CMake will already generate ${CMAKE_BINARY_DIR}/src/mlpack/core/util/gitversion.hpp, which will have this text in it, so I think the only thing to do would be to use a line in the AppVeyor config in order to extract that text and modify the WiX configuration accordingly."